npm-shrinkwrap.json

```diff
@@ -5982,6 +6017,11 @@
       "from": "sax@>=0.6.0",
       "resolved": "https://registry.npmjs.org/sax/-/sax-1.2.2.tgz"
     },
+    "seek-bzip": {
```

Comment:
Does that mean that your udif-reading code won't be able to stream images either?
(apologies if that's a dumb question)